.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,40 @@
+name: Bug report
+description: Report a problem/bug to help us improve
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    attributes:
+      label: "Description of the problem"
+      description: |
+        Please be as detailed as you can when describing an issue.
+        The more information we have, the easier it will be for us to track this down.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: "Full code that generated the error"
+      description: "Provide an example code snippet that reproduces the bug."
+      placeholder: "PASTE CODE HERE"
```

Comment:
Do we want to offer hints on how to create a minimal complete verifiable example (MCWE)? E.g. following xarray's bug report template at https://github.com/pydata/xarray/blob/v2022.06.0/.github/ISSUE_TEMPLATE/bugreport.yml?

Reply:
Yes, MCWE is better.

Reply:
Ok, MCWE hints added in 9a9f454b3d4286fd580da929226a198f59137f51